modules/operators/public_api.ts

```diff
@@ -0,0 +1 @@
+export * from './src/index';
```

Comment:
Is the `public_api.ts` file still needed? `tsconfig.build.json` and `ng-package.json` from `@ngrx/operators` already point to `index.ts`. (Btw, I removed the `public_api` file from `@ngrx/signals`)